$OpenBSD: patch-configure,v 1.9 2012/01/09 12:10:22 dcoppa Exp $

Don't check for libgpod 0.6.x: it's not in our ports tree

--- configure.orig	Thu Dec 29 15:31:00 2011
+++ configure	Mon Jan  9 11:05:54 2012
@@ -626,8 +626,6 @@ GPOD_CFLAGS
 GPOD07_LIBS
 GPOD07_CFLAGS
 GPOD_DEFINES
-GPOD06_LIBS
-GPOD06_CFLAGS
 NOTIFY_DEFINES
 LIBNOTIFY_07_LIBS
 LIBNOTIFY_07_CFLAGS
@@ -868,8 +866,6 @@ NOTIFY_CFLAGS
 NOTIFY_LIBS
 LIBNOTIFY_07_CFLAGS
 LIBNOTIFY_07_LIBS
-GPOD06_CFLAGS
-GPOD06_LIBS
 GPOD07_CFLAGS
 GPOD07_LIBS
 MUSICBRAINZ_CFLAGS
@@ -1598,9 +1594,6 @@ Some influential environment variables:
               C compiler flags for LIBNOTIFY_07, overriding pkg-config
   LIBNOTIFY_07_LIBS
               linker flags for LIBNOTIFY_07, overriding pkg-config
-  GPOD06_CFLAGS
-              C compiler flags for GPOD06, overriding pkg-config
-  GPOD06_LIBS linker flags for GPOD06, overriding pkg-config
   GPOD07_CFLAGS
               C compiler flags for GPOD07, overriding pkg-config
   GPOD07_LIBS linker flags for GPOD07, overriding pkg-config
@@ -9220,86 +9213,6 @@ fi
 if test "x$with_libgpod" != xno; then :
 
 pkg_failed=no
-{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for GPOD06" >&5
-$as_echo_n "checking for GPOD06... " >&6; }
-
-if test -n "$GPOD06_CFLAGS"; then
-    pkg_cv_GPOD06_CFLAGS="$GPOD06_CFLAGS"
- elif test -n "$PKG_CONFIG"; then
-    if test -n "$PKG_CONFIG" && \
-    { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"libgpod-1.0 <= 0.6.0\""; } >&5
-  ($PKG_CONFIG --exists --print-errors "libgpod-1.0 <= 0.6.0") 2>&5
-  ac_status=$?
-  $as_echo "$as_me:${as_lineno-$LINENO}: \$? = $ac_status" >&5
-  test $ac_status = 0; }; then
-  pkg_cv_GPOD06_CFLAGS=`$PKG_CONFIG --cflags "libgpod-1.0 <= 0.6.0" 2>/dev/null`
-else
-  pkg_failed=yes
-fi
- else
-    pkg_failed=untried
-fi
-if test -n "$GPOD06_LIBS"; then
-    pkg_cv_GPOD06_LIBS="$GPOD06_LIBS"
- elif test -n "$PKG_CONFIG"; then
-    if test -n "$PKG_CONFIG" && \
-    { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"libgpod-1.0 <= 0.6.0\""; } >&5
-  ($PKG_CONFIG --exists --print-errors "libgpod-1.0 <= 0.6.0") 2>&5
-  ac_status=$?
-  $as_echo "$as_me:${as_lineno-$LINENO}: \$? = $ac_status" >&5
-  test $ac_status = 0; }; then
-  pkg_cv_GPOD06_LIBS=`$PKG_CONFIG --libs "libgpod-1.0 <= 0.6.0" 2>/dev/null`
-else
-  pkg_failed=yes
-fi
- else
-    pkg_failed=untried
-fi
-
-
-
-if test $pkg_failed = yes; then
-   	{ $as_echo "$as_me:${as_lineno-$LINENO}: result: no" >&5
-$as_echo "no" >&6; }
-
-if $PKG_CONFIG --atleast-pkgconfig-version 0.20; then
-        _pkg_short_errors_supported=yes
-else
-        _pkg_short_errors_supported=no
-fi
-        if test $_pkg_short_errors_supported = yes; then
-	        GPOD06_PKG_ERRORS=`$PKG_CONFIG --short-errors --print-errors "libgpod-1.0 <= 0.6.0" 2>&1`
-        else
-	        GPOD06_PKG_ERRORS=`$PKG_CONFIG --print-errors "libgpod-1.0 <= 0.6.0" 2>&1`
-        fi
-	# Put the nasty error message in config.log where it belongs
-	echo "$GPOD06_PKG_ERRORS" >&5
-
-	{ $as_echo "$as_me:${as_lineno-$LINENO}: Compiling without libgpod 0.6.0 support" >&5
-$as_echo "$as_me: Compiling without libgpod 0.6.0 support" >&6;}
-elif test $pkg_failed = untried; then
-     	{ $as_echo "$as_me:${as_lineno-$LINENO}: result: no" >&5
-$as_echo "no" >&6; }
-	{ $as_echo "$as_me:${as_lineno-$LINENO}: Compiling without libgpod 0.6.0 support" >&5
-$as_echo "$as_me: Compiling without libgpod 0.6.0 support" >&6;}
-else
-	GPOD06_CFLAGS=$pkg_cv_GPOD06_CFLAGS
-	GPOD06_LIBS=$pkg_cv_GPOD06_LIBS
-        { $as_echo "$as_me:${as_lineno-$LINENO}: result: yes" >&5
-$as_echo "yes" >&6; }
-	GPOD_DEFINES="-DHAVE_GPOD -DGPOD_06"
-$as_echo "#define GPOD_ENABLED 1" >>confdefs.h
-
-fi
-
-
-
-
-fi
-
-if test "x$with_libgpod" != xno; then :
-
-pkg_failed=no
 { $as_echo "$as_me:${as_lineno-$LINENO}: checking for GPOD07" >&5
 $as_echo_n "checking for GPOD07... " >&6; }
 
@@ -9377,8 +9290,8 @@ fi
 
 fi
 
-GPOD_CFLAGS="$GPOD06_CFLAGS $GPOD07_CFLAGS"
-GPOD_LIBS="$GPOD06_LIBS $GPOD07_LIBS"
+GPOD_CFLAGS="$GPOD07_CFLAGS"
+GPOD_LIBS="$GPOD07_LIBS"
 
 
 
